notebooks/community/ml_ops/stage3/get_started_with_hpt_pipeline_components.ipynb

```diff
@@ -1,1446 +1,1472 @@
 {
```

Comment:
<p>On the Upload the best model step, add explanation what importer does.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>Done</p>